config/initializers/devise.rb

```diff
@@ -0,0 +1,311 @@
+# frozen_string_literal: true
```

Comment:
This is the default devise configuration. It is created when `rails generate devise:install` is run (as [described in the README](https://github.com/heartcombo/devise#getting-started))

I've not modified this file from the default.